fns-hl7-pipeline/fn-receiver-debatcher/pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>gov.cdc.dataexchange</groupId>
     <artifactId>receiver-debatcher</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>${fn.version}</version>
     <packaging>jar</packaging>
```

Comment:
We are doing the version differently now -- please see the Comments section of the Jira ticket HL7-2454

Reply:
updated, please review

Reply:
updated, please review